src/App.css

```diff
@@ -7,18 +7,36 @@ nav.navbar {
   display: flex;
   justify-content: space-evenly;
   align-items: baseline;
+  margin-left: 10px;
+}
+
+h1.site-title {
+  color: #0290ff;
+}
+
+a.navlink {
+  color: rgb(140, 140, 140);
+  text-decoration: none;
+  margin-left: 15px;
+}
+
+.navlink :hover {
+  color: black;
 }
 
 .item {
   display: flex;
   flex-direction: row;
   justify-content: space-evenly;
+  width: 100%;
 }
 
 .item-left {
   display: flex;
   flex-direction: column;
   justify-content: space-evenly;
+  width: 250px;
+  max-width: 250px;
```

Comment:
- [ ] I would recommend keeping the UI same as the design so that users can see each book in a different section. [ Hint: Give a different color in between each book section].

### Your design 

![Screenshot from 2021-10-02 21-30-11](https://user-images.githubusercontent.com/22434660/135724201-a933e21e-d612-4523-a39a-b12152140f2d.png)

### Given design
![Screenshot from 2021-10-02 21-29-46](https://user-images.githubusercontent.com/22434660/135724198-ed38086b-ad50-4d82-a657-b41e320fbfc1.png)